titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/PlatformSidecar.java

```diff
@@ -2,17 +2,20 @@
 
 import java.util.Objects;
 import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
 
 
 public class PlatformSidecar {
 
-    @Valid
+    @NotNull
+    @Pattern(regexp = "[a-z0-9]([-a-z0-9]*[a-z0-9])?", message = "platform sidecar names name must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character")
```

Comment:
Let's make these regexps consistent with the validation we apply at the InfraAPI level: https://stash.corp.netflix.com/projects/NCE/repos/platform-sidecar-validator/browse/validate.go#231

Reply:
added